---
title: "Incubation Phylogenetic Tree Notebook"
output: html_notebook
---
We used Jin's python script to generate an aligned fasta file with the representative sequences for each OTU from the incubated microcosm soils and amendments. A phylogenetic tree was constructed using this aligned file using FastTree.       

Download from HPCC using SCP
```{bash eval = F}
scp flaterj1@rsync.hpcc.msu.edu:/mnt/research/germs/jared/Incubation/raw/tree.nwk ./tree.nwk
```
```{r eval = F}
if (!requireNamespace("BiocManager", quietly = TRUE))
    install.packages("BiocManager")
BiocManager::install("ggtree", version = "3.8")
BiocManager::install("phyloseq", version = "3.8")
```

```{r}
library(tidyverse)
library(ggtree)

tree <- read.tree("data/tree.nwk")
tree

inc_phy <- readRDS("data/RDS/incubation_physeq_Aug18.RDS")

# Add the tree file to the phyloseq object
inc_phy <- merge_phyloseq(inc_phy, tree)
inc_phy
```
